src/rollbar.php

```diff
@@ -176,6 +176,10 @@ public function __construct($config) {
 
     public function report_exception($exc, $extra_data = null, $payload_data = null) {
         try {
+            if (!$exc instanceof Exception) {
+                throw new Exception('Report exception requires an instance of Exception.');
```

Comment:
I think this is probably the right thing to do, but it is a backwards-incompatible change. Will bump the version number to 0.11.0.